README.md

```diff
@@ -114,24 +122,48 @@ See our [example](./example) for how to use the client package.
 
 ## References
 
-### Response JSON config example
+### Response config example
 ```
 {
-	"Versions": [{
-		"Name": "v1.0.0",
-		"ReleaseDate": "2020-05-30T10:20:00Z",
-		"Tags": ["latest"]
+	"versions": [{
+		"name": "v1.0.0",
+		"releaseDate": "2020-05-30T10:20:00Z",
+		"tags": ["latest"]
 	}]
 }
```

Comment:
I feel I said this before about whether the current client still can decode this new response format correctly? IIRC, it should be YES, it seems the builtin go json decoder.

Reply:
Yeah, go json decoder will still understand the new format. I have tested with Longhorn v1.2.4